vnext/Microsoft.ReactNative.Cxx.UnitTests/ReactContextTest.cpp

```diff
@@ -108,6 +120,70 @@ TEST_CLASS (ReactContextTest) {
     TestCheck(context11 != nullptr);
     TestCheck(nullptr != context11);
   }
+
+  TEST_METHOD(Test_CallJSFunction) {
+    auto reactContextMock = winrt::make_self<ReactContextStub>();
+    ReactContext context{reactContextMock.as<IReactContext>()};
+    context.CallJSFunction(L"module1", L"method1");
+    TestCheckEqual(L"module1", reactContextMock->Module);
+    TestCheckEqual(L"method1", reactContextMock->Method);
+    TestCheckEqual(0u, reactContextMock->Args.AsArray().size());
+
+    context.CallJSFunction(L"module1", L"method1", 5);
+    TestCheckEqual(1u, reactContextMock->Args.AsArray().size());
+    TestCheckEqual(5, reactContextMock->Args[0]);
+
+    context.CallJSFunction(L"module1", L"method1", 14, 17);
+    TestCheckEqual(2u, reactContextMock->Args.AsArray().size());
+    TestCheckEqual(14, reactContextMock->Args[0]);
```

Comment:
consider adding a test case to make sure comparison between signed/unsigned arg and unsigned/signed value works?

Reply:
Could you please provide more details on what type of test it should be? In this context the code that is being tested does not do any comparison.
Are you referring to the multiple signed/unsigned warnings we see today in many unit tests? I am not sure where they came from and why they are 32-bit only...

Reply:
I meant something like `TestCheckEqual(14u, reactContextMock->Args[0])`